### PR TITLE
Travis-native test executions and headless Chrome

### LINKED ISSUE #164: Drop Gizra Drupal Lamp [8h]
https://medium.com/google-developers/how-to-run-travisci-locally-on-docker-822fc6b2db2e gives the ability to test locally.

We could save significant amount of time to use the environment what Travis uses out of the box.

So we could:
 - port all the test / setup scripts from `ci-scripts/docker_files` to work with the native environment
 - drop Zalenium, use headless Chrome to execute test, expose database dump after failed test with Gizra Robot., #162

### PR DESCRIPTION
**Pull request by AronNovak**
will fix #164 

## Status
started to reorganize, with that, surely broke the tests

### CODE REVIEW

File: ci-scripts/test_server.sh [hunk 1]
```diff
@@ -14,88 +14,81 @@ fi
...
-    cd /tmp
-    ! rm gdrive-linux-x64.zip
-    wget https://github.com/prasmussen/gdrive/files/879060/gdrive-linux-x64.zip
-    unzip gdrive-linux-x64.zip
-    chmod +x ./gdrive
-    mkdir ~/.gdrive
-    cp "$TRAVIS_BUILD_DIR"/gdrive-service-account.json ~/.gdrive/
-    GH_COMMENT=/tmp/video_urls
-    echo -n '{
+  echo "One of the containers exited with $code"
```
Comment (amitaibu) [inline ci-scripts/test_server.sh]: I think we can drop the video for now, or at least making it optional. In the end, having the DB before WDIO failed seems to be better, as it's hard to spot the problem with the video.

Comment (AronNovak) [reply ci-scripts/test_server.sh]: I totally agree with that, we can upload the database dump. But if we don't have random demo content, isn't it more useful to upload the dump after the failure (examine field values, DB log entries, etc)?

Comment (amitaibu) [reply ci-scripts/test_server.sh]: 👍

**Comment by AronNovak** [comment]
Now before going further, I make sure that I can follow https://medium.com/google-developers/how-to-run-travisci-locally-on-docker-822fc6b2db2e and indeed, I am able to simulate Travis at local with ease.
**1.05/4h**

**Comment by AronNovak** [comment]
Current status:
image

Travis.yml compilation fails.

**Comment by AronNovak** [comment]
So actually that blogpost is outdated, it's impossible to reproduce all those steps, going to refer the official documentation for the same purpose:
https://docs.travis-ci.com/user/common-build-problems/#Troubleshooting-Locally-in-a-Docker-Image

**Comment by AronNovak** [comment]
image

Not yet successful to execute the compiled YML file (`travis compile` now successfully generates a shell script, but it fails like above).

**Comment by AronNovak** [comment]
A bit further:

**Comment by AronNovak** [comment]
image

Now in the native Travis environment, the Drupal installation happens and the site is up and running via the native PHP webserver.

**Comment by AronNovak** [comment]
image

Simpletest is executed successfully in the native environment

**Comment by AronNovak** [comment]
image

Except WDIO, all the executions are restored.

**Comment by AronNovak** [comment]
@amitaibu ready for a final review :)

**Comment by amitaibu** [comment]
Merged, thanks!

**Pull request merged**